plugin-server/src/cdp/services/hogflows/billing-utils.ts

```diff
@@ -0,0 +1,26 @@
+import { CyclotronJobInvocation, CyclotronJobInvocationResult } from '~/cdp/types'
+
+type HogFlowBillingMetricData = {
+    invocation: CyclotronJobInvocation
+    billingMetricType: 'fetch' | 'email'
```

Comment:
`sms` and `push` is missing. should this be aligned with `metric_kind` in `plugin-server/src/cdp/types.ts`?